ipywidgets/static/widgets/js/layout.js

```diff
@@ -104,7 +103,7 @@ define([
          */
         handleChange: function(trait, value) {
             this.displayed.then(_.bind(function(parent) {
-                if (parent && value) {
```

Comment:
I am reverting a change I did in #285. Otherwise, one cannot set a property back to `''`.